### PR TITLE
Add support for plain control types

### PR DESCRIPTION
**Pull request by kant2002**
Closes: #1193

### CODE REVIEW

**Comment by cartermp** [comment]
Failed a formatting check: `Some files need formatting, run `dotnet fake build -t Format` to format them`

**Comment by cartermp** [comment]
Yeah, I don't know of a way to make this non-breaking, so it's fine. I doubt most people rely on this type and pattern match on it.

Release schedule is whenever I cut a release, heh. I'll probably do that soon once this lands, assuming CI gets green.

**Comment by kant2002** [comment]
I think I done with formatting.

> Release schedule is whenever I cut a release, heh. I'll probably do that soon once this lands, assuming CI gets green

I like this spirit. Now I can take a look, and shop for fun up for grabs.

**Comment by kant2002** [comment]
Let me wrestle with Fake. 😢

File: .github/workflows/pull-requests.yml [hunk 1]
```diff
@@ -20,6 +21,8 @@ jobs:
     - name: Restore packages
       run: dotnet paket restore
     - name: Build and test (Release)
+      env:
+        FAKE_DETAILED_ERRORS: true
```
Comment (cartermp) [inline .github/workflows/pull-requests.yml:25]: nice, agreed

**Review by cartermp — APPROVED** [review]
Alrighty, re-review looks good -- I'll merge this puppy in and see what all else is needed for another update this week.

**Comment by kant2002** [comment]
Most of issues which I have is that I being unable to make Fake run. So I really should either fix Fake somehow or beg for babysitting me by somebody.

**Comment by kant2002** [comment]
@cartermp can I have some fresh and tasty release please ? )

**Comment by cartermp** [comment]
Done!

**Comment by cartermp** [comment]
https://github.com/fsprojects/FSharp.Data/releases/tag/6.3.0

**Comment by kant2002** [comment]
Thank you!!!!

**Pull request merged**